src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,18 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        StringBuilder sb = new StringBuilder();
```

Comment:
let's create more understandable name of the variable instead of `sb`

Reply:
Changed name of variable sb to myStringBuilder